app/views/handlers/question.py

```diff
@@ -128,6 +129,11 @@ def get_context(self):
                 )
             )
 
+        if self.form.errors or self.form.question_errors:
+            self.page_title = gettext("Error: {page_title}").format(
+                page_title=self.page_title
+            )
+
```

Comment:
We should have a test for this. We use integration tests to test the handlers, so I think the best place for it would be in the `tests/integration/questionnaire` folder. As we will be doing more work to make the page titles consistent, it's probably worth creating a new `test_page_titles.py` test file in that folder.